comptests/prov/test_prov_k8s_cortx_upgrade.py

```diff
@@ -298,6 +296,11 @@ def test_37514(self):
             resp = str(resp, 'UTF-8')
         resp = "".join(resp).replace("\\n", "\n")
         assert_utils.assert_in("Invalid argument provided", resp)
+        LOGGER.info("Remove upgrade pid file for next TC.")
```

Comment:
This can be either added to setup method (as a precheck) or teardown method.

Reply:
Thanks. Let me add it.

Reply:
Done. Have put that login in teardown method.